lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -103,7 +103,7 @@ protected function doSave($id, $data, $lifeTime = 0)
         $code   = sprintf('<?php return %s;', $value);
 
         $ret = (file_put_contents($filename, $code) !== false);
-        chmod($filename, 0664);
+        @chmod($filename, 0664);
```

Comment:
What may be the outcome (later on) if there is a failure in executing this operation? Doesn't feel right to just silence it without handling it.